legate/core/operation.py

```diff
@@ -293,13 +330,21 @@ def _add_communicators(self, launcher, launch_domain):
 
 
 class AutoTask(Task):
-    def __init__(self, context, task_id, mapper_id=0, op_id=0):
-        Task.__init__(self, context, task_id, mapper_id, op_id)
-
-    def launch(self, strategy):
+    def __init__(
+        self,
+        context: Context,
+        task_id: int,
+        mapper_id: int = 0,
+        op_id: int = 0,
+    ) -> None:
+        super().__init__(context, task_id, mapper_id, op_id)
+
+    def launch(self, strategy: Strategy) -> None:
         launcher = TaskLauncher(self.context, self._task_id, self.mapper_id)
 
-        def get_requirement(store, part_symb):
+        def get_requirement(
+            store: Store, part_symb: PartSym
+        ) -> tuple[Any, Literal[1, 0], StorePartition]:
```

Comment:
Ok. so you used the type `Literal[1, 0]` here as well. why can't it be just `int`?

Reply:
https://github.com/nv-legate/legate.core/pull/189/commits/99257dbf2be5778ac6fe2febfa88cf35ab1e2673